unfoldingWord/translationCore
# Default CSV Export Location

#### This pull request addresses:
This PR addresses issues in #1693 


#### How to test this pull request:
To test try exporting to different locations and see if it saves the last place.
The first attempt to save should also be in the Documents folder (Win & Mac)


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1735)
